### PR TITLE
Remove DataGridViewBand._bandIsRow

### PR DESCRIPTION
**Pull request by hughbe**
Just use the `IsRow` setter and simplify the check

Follow-up to #1390

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/1427)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewBand.cs [hunk 1]
```diff
@@ -421,14 +417,14 @@ internal int MinimumThickness
             }
         }
 
-        internal PropertyStore Properties => _propertyStore;
+        internal PropertyStore Properties { get; private set; } = new PropertyStore();
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewBand.cs]:
```suggestion
        /// <summary>
        /// Contains all properties that are not always set.
        /// </summary>
        internal PropertyStore Properties { get; private set; } = new PropertyStore();
```

**Comment by RussKie** [comment]
Please merge the last commit with 31445d38841938520d22ec1966f816766a2ae2a8 and it is good to go

**Comment by RussKie** [comment]
Consider using [fixup commits](https://www.mikulskibartosz.name/git-fixup-explained/), they make life easier 😉

**Comment by hughbe** [comment]
Aye I do use them, but unfortunately I think I have to force push each time ?

**Comment by RussKie** [comment]
No, fixup commits stay on top - so you push them as normal
And when such time comes to clean up the work - do an interactive rebase and only then force push

**Pull request merged**